### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Autoc0diq**

### CODE REVIEW

File: Example/ExampleApp/AUTTripListController.m [hunk 1]
```diff
@@ -15,7 +15,7 @@ @interface AUTTripListController ()
 
 @property (readwrite, nonatomic, strong) AFHTTPRequestOperation *currentRequest;
 
-@property (readwrite, nonatomic, strong) NSURL *nextPage;
+@property (readwrite, nonatomic, strong) NSURL *nextPageUrl;
```
Comment (robb) [inline Example/ExampleApp/AUTTripListController.m]: this should be `nextPageURL`

**Comment by robb** [comment]
I feel like we might as well pull in `EXTScope` at this point, what do you think?

**Comment by erichoracek** [comment]
@robb make it an explicit dependency or just pull in the macro?

**Comment by robb** [comment]
Just copy it in and prefix `ext_executeCleanupBlock`

**Comment by jspahrsummers** [comment]
libextobjc is available as subspecs, so you could do:

```
pod "libextobjc/EXTScope"
```

**Comment by erichoracek** [comment]
@jspahrsummers have there ever been any breaking changes to `libextobjc/EXTScope` that would force us to rely on a subset of versions? If so, it might be a bit of trouble brining it in if consumers are already using it (as they could have a conflict). If not, I think it would make sense to bring it in.

To be fair, RAC copies weakify/strongify in instead of depending on `libextobjc/EXTScope`, correct? :wink:

**Comment by robb** [comment]
We want to move to Carthage, eventually, too…

**Comment by erichoracek** [comment]
If you rebase this on master you should be able to get the build working.

**Comment by jspahrsummers** [comment]
> have there ever been any breaking changes to `libextobjc/EXTScope` that would force us to rely on a subset of versions?

Nah, I don't think so. And it shouldn't be changing anytime soon either.

> To be fair, RAC copies weakify/strongify in instead of depending on `libextobjc/EXTScope`, correct? :wink: 

Yes, because RAC doesn't use CocoaPods. :wink:

**Comment by erichoracek** [comment]
@jspahrsummers :cool:. Since that is the case I don't see any issue with depending on it with no version requirements as part of this commit by adding the following to `AutomaticSDK.podspec`:
```
s.dependency 'libextobjc/EXTScope'
```
Once less thing that we have to maintain.

**Pull request merged**